models/scores.go

```diff
@@ -5,14 +5,16 @@ type Scores struct {
 	URL      string   `json:"url"`
 	Score    int      `json:"score"`
 	Messages []string `json:"messages"`
+	Cert     *Cert    `json:"certificate_details"`
```

Comment:
I suggest we have the certificate details separately maybe? Ideally the certificate details might have different values  and rather than sending it as one whole thing, we could have a struct with just the cert details in the models.

I feel the scores struct should store only the final score sent to the user. any data that needs to be appended to the final response, should be done in the request handler.

Reply:
Maybe a struct named Response that combines both Scores and Cert?